sdcm/cluster.py

```diff
@@ -2045,14 +2045,24 @@ def _parse_bench_summary(lines):
             if line.startswith('Results'):
                 enable_parse = True
                 continue
-            if line.startswith('Latency:'):
+            if line.startswith('Latency:') or ':' not in line:
                 continue
             if not enable_parse:
                 continue
 
             split_idx = line.index(':')
             key = line[:split_idx].strip()
             value = line[split_idx + 1:].split()[0]
+            try:
+                value = value.decode('utf-8').encode('utf-8').replace('\xc2\xb5s', 'mcs')
```

Comment:
This looks ugly, sorry :)
What is the reason for that?

Reply:
agree, I'll replace it by something more readable